composer.json

```diff
@@ -20,8 +20,9 @@
         "symfony/class-loader": "~2.8|~3.0|~3.2"
     },
     "require-dev": {
-        "corneltek/phpunit-testmore": "dev-master",
-        "satooshi/php-coveralls": "^1"
+        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.5",
+        "satooshi/php-coveralls": "^1",
+        "ext-intl": "*"
```

Comment:
Adding the `intl` extension checking because it should use this extension during tests running.

Otherwise it will get the following exception when running tests:

```
1) DateFormatCellTest::testDateFormat
Error: Class 'IntlDateFormatter' not found

/home/localadmin/CLIFramework/tests/CLIFramework/Component/Table/DateFormatCellTest.php:10
phpvfscomposer:///home/localadmin/CLIFramework/vendor/phpunit/phpunit/phpunit:60
```